pkg/backend/display/progress.go

```diff
@@ -247,8 +247,10 @@ func ShowProgressEvents(op string, action apitype.UpdateKind, stack tokens.Name,
 
 	var renderer progressRenderer
 	if isInteractive {
-		renderer = newInteractiveRenderer(term, opts)
+		printOpenInBrowser(term, opts, permalink)
+		renderer = newInteractiveRenderer(term, permalink, opts)
 	} else {
+		printViewLive(stdout, opts, permalink)
```

Comment:
Can we add a comment on why we're keeping `View Live` for non-interactive? (Mainly for back-compat with automation API).

Reply:
Yeah--I added a comment to the commit, but I'll add a comment in the code as well.